web/package.json

```diff
@@ -43,6 +43,7 @@
     "@patternfly/react-core": "5.4.0",
     "@patternfly/react-icons": "^5.4.0",
     "@patternfly/react-table": "4.113.0",
+    "@patternfly/react-templates": "^1.1.1",
```

Comment:
Do we need this templates dependency? it seems this is only for templates to get started on using patternfly

Reply:
We need the  "@patternfly/react-templates" because we are importing `<SimpleSelect />` [[1](https://github.com/openshift/monitoring-plugin/blob/1e2c229263e6a6fdce3b025afecf3300bc81e461/web/src/components/dropdown-poll-interval.tsx#L39)].

`<SimpleSelect />` is a template, I can copy the code into the `/monitoring-plugin` and remove the dependency. WDYT?

Reply:
yes, we might not need all the code but I think is better to use the core components

Reply:
Update this in the latest commit.